### PR TITLE
RHDEVDOCS-4931 - Logging 5.6.3 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.10+

Issue: [RHDEVDOCS-4931](https://issues.redhat.com//browse/RHDEVDOCS-4931)

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cluster-logging-rn-5.6.3.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-6-3_{context}"]
+= Logging 5.6.3
+This release includes link:https://access.redhat.com/errata/RHSA-2023:0932[OpenShift Logging Bug Fix Release 5.6.3].
+
+[id="openshift-logging-5-6-3-bug-fixes"]
+== Bug fixes
+* Before this update, the operator stored gateway tenant secret information in a config map. With this update, the operator stores this information into a secret. (link:https://issues.redhat.com/browse/LOG-3717[LOG-3717])
```
Comment (rolfedh) [inline modules/cluster-logging-rn-5.6.3.adoc:9]: Maybe "in a secret" or "as a secret"?
```suggestion
* Before this update, the operator stored gateway tenant secret information in a config map. With this update, the operator stores this information in a secret. (link:https://issues.redhat.com/browse/LOG-3717[LOG-3717])
```

**Review by rolfedh — APPROVED** [review]
Nice work. Offered a minor suggestion. Approved to expedite.

**Review by jcantrill — COMMENTED** [review]
/lgtm

**Pull request merged**